### ISSUE TITLE
Modelo de casos de uso

### ISSUE THREAD

**Issue by seguijoaquin** [seguijoaquin/7509-AnInfo#1]
Genio Fer por arrancar con el diagrama

Tenemos primero, identificación de actores:
-Cliente
-Tesoreria
-Prestadores

No haría falta un actor más??
Sea Ventas o Agencia  o Empresa o lo que sea, cuya responsabilidad sea confeccionar el contrato, entregar el talonario con todas las órdenes de servicio, comunicarse con los prestadores, y negociar las modificaciones de las reservas con los clientes?

**Comment by NKeklikian** [comment]
El tema es que todas esas actividades no usan el sistema, serían actores interactuando, entonces no sé si ponerlas como casos de uso. Ventas lo terminamos sacando con el argumento de "es parte del sistema". Igual si a vos se te ocurre un diagrama que quede mejor ponelo y lo llevamos

**Comment by seguijoaquin** [comment]
Claro, serían más bien interacciones dentro del sistema o entre actores
